O/open62541/build_tarballs.jl

```diff
@@ -44,7 +45,8 @@ products = [
 ]
 
 # Dependencies that must be installed before this package can be built
-dependencies = Dependency[
+dependencies = [
+    Dependency("MbedTLS_jll"),
```

Comment:
> Also, @giordano is mbedtls still a stdlib where version compats have to be kept in line with Julia versions?

Yes.  From Julia [v1.8](https://github.com/JuliaLang/julia/blob/5544a0fab7648cfa61fe79cd557a7504a92ec1b5/stdlib/MbedTLS_jll/Project.toml#L3) to [current `master`](https://github.com/JuliaLang/julia/blob/acddc62c1acca478d39c73f8c248d519ed4cfe93/stdlib/MbedTLS_jll/Project.toml#L3) you can use
```suggestion
    Dependency("MbedTLS_jll"; compat="~2.28"),
```
but this also means you ***must*** change the julia compat below, which also implies you ***must*** change the version number of the package.